comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -158,6 +163,7 @@
       <packagereq type="default">rubygem-ruby-hmac-doc</packagereq>
       <packagereq type="default">rubygem-satyr-doc</packagereq>
       <packagereq type="default">rubygem-sequel-doc</packagereq>
+      <packagereq type="default">tfm-rubygem-recursive-open-struct</packagereq>
```

Comment:
this looks out of place here (sorting wise)

Reply:
moved to line 136:
```
      <packagereq type="default">tfm-rubygem-rainbow</packagereq>
      <packagereq type="default">tfm-rubygem-recursive-open-struct</packagereq>
      <packagereq type="default">tfm-rubygem-rprogram</packagereq>
```